app_wshwsim/sl_rf_driver.c

```diff
@@ -41,7 +44,17 @@ static uint8_t rf_mac_address[8];
 static int8_t rf_driver_id = (-1);
 static bool data_request_pending_flag = false;
 static uint16_t channel = 0;
+static bool rf_state = false;
+static int tx_time = 0;
+static bool cca_enabled = false;
 
+#ifdef CHANNEL_HOPPING_ON
+struct data_content {
+    uint8_t *data_ptr;
+    uint16_t data_len;
+    uint8_t tx_handle;
+} tx_data_content;
+#endif /* CHANNEL_HOPPING_ON */
```

Comment:
Does it make sense to declare `tx_data_content` globally? It seems it can a local variable of `phy_rf_tx()`, doesn't?